src/coreComponents/constitutive/solid/SolidBase.hpp

```diff
@@ -606,6 +606,9 @@ class SolidBase : public constitutive::ConstitutiveBase
   /// Save state data in preparation for next timestep
   virtual void saveConvergedState() const override;
 
+  /// Resets some quantities in the solid model after the equilibration step (relevant for plasticity models only)
+  virtual void applyPostEquilibrationStep() const {};
```

Comment:
Do you plan to add more feature specific to plasticity models only?
If so, you could implement a `PlasticSolid`?

Reply:
Should the `post-equilibration` step be applied for each individual mechanics model or just turned on in the `coupledSolidBase.hpp`?

Reply:
Now this is implemented in a better way: there is a `disableInelasticity()` function in `SolidBase` to turn on/off a flag. This flag is only used by the plastic models to turn off plasticity during the equilibration step, and turn it on again after the equilibration step.